index.js

```diff
@@ -52,6 +52,15 @@ function getOptions(addonContext) {
     }
   }
 
+  // add babel-plugins from the registry to the options.plugins
+  var pluginWrappers = addonContext.registry.load('babel-plugin');
```

Comment:
Hmm, using `addonContext.parent.registry.load('babel-plugin') gives me a `Cannot read property 'load' of undefined`.